Tasks/ManualValidationV1/task.json

```diff
@@ -0,0 +1,69 @@
+{
+    "id": "2003AD3A-104A-451A-9238-60474AB294E6",
+    "name": "ManualValidation",
+    "friendlyName": "Manual validation",
+    "description": "[PREVIEW] Pause a pipeline run to wait for manual interaction. Works only with YAML pipelines.",
+    "helpUrl": "https://aka.ms/manual-validation",
+    "helpMarkDown": "[Learn more about this task](https://aka.ms/manual-validation)",
+    "category": "Deploy",
+    "visibility": [
+        "Build"
+    ],
+    "author": "Microsoft Corporation",
+    "runsOn": [
+        "Server"
+    ],
+    "version": {
+        "Major": 1,
+        "Minor": 240,
+        "Patch": 1
+    },
+    "preview": true,
+    "inputs": [
+        {
+            "name": "notifyUsers",
+            "type": "multiLine",
+            "label": "Notify users",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Send a manual validation pending email to specific users (or groups). Only users with queue build permission can act on a manual validation."
+        },
+        {
+            "name": "approvers",
+            "type": "multiLine",
+            "label": "Approvers",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Specify users/groups/project teams to act on a manual validation."
```

Comment:
Could you please add the description how the format should look like (separator etc.)?

Reply:
updated the description